src/azure-cli/azure/cli/command_modules/keyvault/_params.py

```diff
@@ -287,7 +287,7 @@ class CLISecurityDomainOperation(str, Enum):
                 try:
                     if cmd in ['list']:
                         c.extra('identifier', options_list=['--id'],
-                                help='Id of the Vault or HSM. '
+                                help='Full URI of the Vault or HSM. '
```

Comment:
Personally I think it is anti-convention to name it "URI" in Azure Portal (https://github.com/Azure/azure-cli/issues/17399#issuecomment-969876246), even though it has `https://` prefix.

This was discussed previously at #13067. 

Also see https://danielmiessler.com/study/difference-between-uri-url/

> If the protocol (https, ftp, etc.) is either present or implied for a domain, you should call it a URL—even though it’s also a URI.